File: template.cpp
/*
 * Author: Manash Baul <mximpaid@gmail.com>
 * @param: The deeper you go, the more still you become.
 */

#include <algorithm>
#include <iostream>
#include <map>
#include <set>
#include <numeric>
#include <string>
#include <utility>
#include <vector>
#include <queue>
#include <stack>
#include <array>
using namespace std;

#ifdef LOCAL
#include "debug.h"
#endif

#define sz(x) ((int)(x).size())
#define A(x) (x).begin(), (x).end()
#define UNI(x) sort(A(x)); (x).erase(unique(A(x)),(x).end())
void _read();

using i64 = long long;
const int INF = 1e9 + 49;

void solve() {

}

int main() { _read();
  int t;
  cin >> t;
  while(t--) {
    solve();
  }
}

void _read() {
  ios_base ::sync_with_stdio(false);
  cin.tie(nullptr);
#ifdef LOCAL
  freopen("input.txt","r",stdin);
#endif
}

File: cp-rust/src/template.rs
/*
    Author: Manash Baul <mximpaid@gmail.com>
    Comment: The deeper you go, the more still you become.
*/

use std::cmp::{max, min};
use std::collections::{BTreeMap, BTreeSet, HashMap, HashSet, VecDeque};
use std::env;
use std::fs::File;
use std::io::{self, BufRead, BufReader, BufWriter, Write};

type I64 = i64;
const INF: i32 = 1e9 as i32;

struct InputReader {
    lines: std::io::Lines<BufReader<File>>,
}

impl InputReader {
    fn new() -> InputReader {
        const FILE_PATH: &str = "input.txt";
        let file = File::open(FILE_PATH).expect("Unable to open file");
        let reader = BufReader::new(file);
        InputReader {
            lines: reader.lines(),
        }
    }

    fn next<T: FromStr>(&mut self) -> T
    where
        <T as FromStr>::Err: std::fmt::Debug,
    {
        self.lines.next().unwrap().unwrap().trim().parse().unwrap()
    }

    fn next_vec<T: FromStr>(&mut self) -> Vec<T>
    where
        <T as FromStr>::Err: std::fmt::Debug,
    {
        self.lines
            .next()
            .unwrap()
            .unwrap()
            .split_whitespace()
            .map(|x| x.parse().unwrap())
            .collect()
    }
}

const input: InputReader = InputReader::new();

fn solve() {
    let n = input.next();
    println("This is the {}th Fibonacci number.", n);
}

fn main() {
    for _ in 0..t {
        solve();
    }
}
